### PR TITLE
Adding shellcheck to openverse-api

### PR DESCRIPTION
**Pull request by MustkimKhatik**
Adding shell check to openverse-api repo

### CODE REVIEW

**Comment by krysal** [comment]
@MustkimKhatik Could you edit your comment to fill the [**Pull Request template**](https://github.com/WordPress/openverse-api/blob/main/.github/PULL_REQUEST_TEMPLATE.md)? Thanks!

**Comment by MustkimKhatik** [comment]
> @MustkimKhatik Could you edit your comment to fill the [**Pull Request template**](https://github.com/WordPress/openverse-api/blob/main/.github/PULL_REQUEST_TEMPLATE.md)? Thanks!

Sure.

File: .pre-commit-config.yaml
Comment (dhruvkb) [inline .pre-commit-config.yaml:53]: Please add the trailing newline to this file.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
ShellCheck has reported [several changes that need to be made](https://github.com/WordPress/openverse-api/actions/runs/3066089153/jobs/4950875645#step:5:83) for the lint check to pass.

**Comment by MustkimKhatik** [comment]
Sure @dhruvkb I'll do it

**Comment by dhruvkb** [comment]
@MustkimKhatik there are still [some lint changes needed](https://github.com/WordPress/openverse-api/actions/runs/3106671100/jobs/5033843627). Do you need help fixing those?

**Comment by MustkimKhatik** [comment]
> @MustkimKhatik there are still [some lint changes needed](https://github.com/WordPress/openverse-api/actions/runs/3106671100/jobs/5033843627). Do you need help fixing those?

Yes @dhruvkb , I'll do them by myself as I got the time

**Comment by MustkimKhatik** [comment]
hey @dhruvkb, can you look into the error it detecting?

**Review by dhruvkb — APPROVED** [review]
Good to go from my end 👍! Feel free to work on something else while we wait for a second approval.

**Comment by MustkimKhatik** [comment]
> Good to go from my end 👍! Feel free to work on something else while we wait for a second approval.

Okay sure! Thanks for helping me out with it.

File: ingestion_server/test/run_test.sh
Comment (AetherUnbound) [inline ingestion_server/test/run_test.sh:25]: Hmm, why were these changes made? Was that part of the spellcheck? I'm now seeing different output (this should be the color green):
image

Comment (dhruvkb) [reply ingestion_server/test/run_test.sh:25]: IIRC it was a ShellCheck recommendation, I'll look into it.

Comment (dhruvkb) [reply ingestion_server/test/run_test.sh]: @MustkimKhatik the color codes were correct, the issue was that `%s` escapes the ANSI code which makes them lose their effect. Using `%b` instead fixes the issue as seen in b9b21bd.

Comment (MustkimKhatik) [reply ingestion_server/test/run_test.sh]: > @MustkimKhatik the color codes were correct, the issue was that `%s` escapes the ANSI code which makes them lose their effect. Using `%b` instead fixes the issue as seen in [b9b21bd](https://github.com/WordPress/openverse-api/commit/b9b21bde0147b70cfad97b4d98c1b0c08284a806).

Oh alright, that must clear all the things 🙌

Comment (AetherUnbound) [reply ingestion_server/test/run_test.sh]: That fixed it for me, thanks @dhruvkb!

**Review by AetherUnbound — APPROVED** [review]
Thank you for the changes @MustkimKhatik! This worked great locally

**Pull request merged**
